configs/bisenetv1/bisenetv1_r50-d32_lr5e-3_in1k-pre_4x4_512x512_160k_coco-stuff164k.py

```diff
@@ -0,0 +1,7 @@
+_base_ = './bisenetv1_r50-d32_lr5e-3_4x4_512x512_160k_coco-stuff164k.py'
```

Comment:
pretrain should before lr.
Please refer to the config name manual and rename it.